backend/api/Areas/Project/Mapping/Dispose/BuildingMap.cs

```diff
@@ -11,48 +13,45 @@ public void Register(TypeAdapterConfig config)
-                .Map(dest => dest.BuildingOccupantType, src => src.BuildingOccupantType == null ? null : src.BuildingOccupantType.Name)
+                .Map(dest => dest.BuildingOccupantType, src => src.GetOccupantType())
                 .Map(dest => dest.BuildingPredominateUseId, src => src.BuildingPredominateUseId)
-                .Map(dest => dest.BuildingPredominateUse, src => src.BuildingPredominateUse == null ? null : src.BuildingPredominateUse.Name)
+                .Map(dest => dest.BuildingPredominateUse, src => src.GetPredominateUse())
                 .Map(dest => dest.BuildingTenancy, src => src.BuildingTenancy)
                 .Map(dest => dest.BuildingFloorCount, src => src.BuildingFloorCount)
                 .Map(dest => dest.LeaseExpiry, src => src.LeaseExpiry)
                 .Map(dest => dest.OccupantName, src => src.OccupantName)
                 .Map(dest => dest.RentableArea, src => src.RentableArea)
-                .Map(dest => dest.LandArea, src => src.Parcel != null ? src.Parcel.LandArea : 0)
-                .Map(dest => dest.Zoning, src => src.Parcel != null ? src.Parcel.Zoning : "")
-                .Map(dest => dest.ZoningPotential, src => src.Parcel != null ? src.Parcel.ZoningPotential : "")
+                .Map(dest => dest.LandArea, src => src.Parcels.FirstOrDefault() != null ? src.Parcels.FirstOrDefault().Parcel.LandArea : 0)
+                .Map(dest => dest.Zoning, src => src.Parcels.FirstOrDefault() != null ? src.Parcels.FirstOrDefault().Parcel.Zoning : "")
+                .Map(dest => dest.ZoningPotential, src => src.Parcels.FirstOrDefault() != null ? src.Parcels.FirstOrDefault().Parcel.ZoningPotential : "")
                 .Map(dest => dest.IsSensitive, src => src.IsSensitive)
                 .Map(dest => dest.Evaluations, src => src.Evaluations)
                 .Map(dest => dest.Fiscals, src => src.Fiscals)
                 .Inherits<Entity.BaseEntity, BaseModel>();
 
             config.NewConfig<Model.BuildingModel, Entity.Building>()
                 .Map(dest => dest.Id, src => src.Id)
-                .Map(dest => dest.ParcelId, src => src.ParcelId)
-                .Map(dest => dest.LocalId, src => src.LocalId)
+                .Map(dest => dest.Parcels, src => new [] { new Entity.ParcelBuilding() { ParcelId = src.ParcelId, BuildingId = src.Id } })
                 .Map(dest => dest.ProjectNumber, src => src.ProjectNumber)
                 .Map(dest => dest.ClassificationId, src => src.ClassificationId)
                 .Map(dest => dest.Classification, src => src.Classification)
                 .Map(dest => dest.AgencyId, src => src.AgencyId)
                 .Map(dest => dest.Description, src => src.Description)
-                .Map(dest => dest.Latitude, src => src.Latitude)
-                .Map(dest => dest.Longitude, src => src.Longitude)
+                .Map(dest => dest.Location, src => new NetTopologySuite.Geometries.Point(src.Longitude, src.Latitude) { SRID = 4326 })
```

Comment:
Not a deal breaker, but it be nice to create Geometry Extension that returns a Point object with SRID=4326 for reusability

Reply:
I'll add a commit to clean this up.